biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -104,10 +113,17 @@ subroutine IsItLoggingTime(is_master,currentSite)
       logging_time = .false.
       icode = int(logging_event_code)
 
+      ! this is true for either hlm harvest or fates logging
```

Comment:
How is it true for both those cases? Is it set elsewhere?

Reply:
To clarify, hlm_use_logging is set to true if hlm harvest is active or if fates logging is active  (based on namelist inputs). This is so that just one variable is used to determine whether today is a logging/harvest day or not.
The relationship between hlm harvest and logging is this:
Hlm harvest uses hlm input timeseries of harvest prescription to drive the fates logging module, so if hlm harvest is on, then fates logging is automatically turned on.
The fates logging module can still be used on its own by setting its namelist option to true (use_fates_logging), and not setting the hlm do_harvest namelist option (leaving as false).

Reply:
Ah yes. I think I understood this more as I went through the code...